keyboards/viktus/sp_mini/config.h

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+Copyright 2015 Jack Humbert
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x5644
+#define PRODUCT_ID      0x534D
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Viktus_Design
+#define PRODUCT         SP_Mini
+//#define VIAL_KEYBOARD_UID {0xC5, 0x02, 0x78, 0x7D, 0x10, 0xBF, 0x5C, 0x55}
+
+/* key matrix size */
+// Rows are doubled-up
+#define MATRIX_ROWS 10
+#define MATRIX_COLS 8
+
+
+// wiring of each half
+#define MATRIX_ROW_PINS { F0, B5, B4, D7, D6 }
+#define MATRIX_COL_PINS { B6, C6, C7, D4, D2, D3, D5 } // no B7 on left hand
+#define MATRIX_ROW_PINS_RIGHT { F0, B5, B4, D7, D6 }
+#define MATRIX_COL_PINS_RIGHT { B6, C6, C7, D4, D2, D3, D5, B7 }
+
+
+#define DIODE_DIRECTION COL2ROW
+
+/* Split Defines */
+#define SPLIT_USB_DETECT
+#define MASTER_LEFT
+#define USE_I2C
+
+/* serial.c configuration for split keyboard */
+//#define SOFT_SERIAL_PIN D1
+
+
+/* define if matrix has ghost */
+//#define MATRIX_HAS_GHOST
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCE 5
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
```

Comment:
```suggestion
```

Reply:
Is this define being removed because it's deprecated, or just because it's not commonly used?

Reply:
It's not commonly used. And on an atmega32u4 MCU, it can be very tight.  So removing a feature that is rarely used and increases firmware size is probably for the best.

Reply:
gotcha